Dockerfile

```diff
@@ -0,0 +1,39 @@
+# Use the official Alpine-based Golang image as the build stage
+FROM golang:1.18-alpine AS builder
+
+# Set the working directory
+WORKDIR /go/src/github.com/quackduck/devzat
+
+# Copy the project source code into the container
+COPY . .
+
+# Install project dependencies
+RUN go mod download
+
+# Build the project
+RUN go build -o devzat .
+
+
+
+
+# Use the latest Alpine Linux image as the base for the final image
+FROM alpine:latest
+
+# Install necessary tools
+RUN apk add --no-cache openssh-server
+
+# Copy the compiled binary to the final image
+COPY --from=builder /go/src/github.com/quackduck/devzat /usr/local/bin/devzat
+
+# Generate the SSH host key
+RUN ssh-keygen -qN '' -f /etc/ssh/ssh_host_rsa_key
```

Comment:
If i understand Docker correctly this would mean for a published Docker-Image everyone would have the same key. The key should either be generated by Devzat on startup or by the user themselves